client/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
```

Comment:
Do we need multiple .gitignore files for each directory? Couldn't we have one gitignore file in the root directory that also references the sub-directories? https://git-scm.com/docs/gitignore

Reply:
Yes! Was outdated but now fixed with cache removed as well